{{ cookiecutter.package_name }}/.github/workflows/ci.yml

```diff
@@ -0,0 +1,98 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - 'main'
+      - '*.*'
+      - '!*backport*'
+    tags:
+      - 'v*'
+      - '!*dev*'
+      - '!*pre*'
+      - '!*post*'
+  pull_request:
+  # Allow manual runs through the web UI
+  workflow_dispatch:
+
+concurrency:
+  group: {{ '${{ github.workflow }}-${{ github.ref }}' }}
+  cancel-in-progress: true
+
+jobs:
+  core:
+    uses: OpenAstronomy/github-actions-workflows/.github/workflows/tox.yml@main
+    with:
+      submodules: false
+      coverage: codecov
+      toxdeps: tox-pypi-filter
+      envs: |
+        - linux: py312
+    secrets:
+      CODECOV_TOKEN: {{ '${{ secrets.CODECOV_TOKEN }}' }}
+
+  sdist_verify:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-python@v5
+        with:
+          python-version: '3.10'
```

Comment:
In future, this being one variable, we can change would be nice.

Reply:
I have added a commit which addresses this. Have a look and see.

I decided not to add it as  cookiecutter variable as we shouldn't need to have it vary repo-by-repo.

Reply:
I agree